libraries/standard/mqtt/src/iot_mqtt_operation.c

```diff
@@ -109,6 +109,37 @@ static bool _checkRetryLimit( _mqttOperation_t * pOperation );
  */
 static bool _scheduleNextRetry( _mqttOperation_t * pOperation );
 
+/**
+ * @brief Send a PINGREQ packet for a keep-alive operation.
+ *
+ * @param[in] pMqttConnection The associated MQTT connection.
+ *
+ * @return `true` if the operation was successful; `false` otherwise.
+ */
+static bool _sendKeepAlivePingReq( _mqttConnection_t * pMqttConnection );
+
+/**
+ * @brief Schedule a callback for a completed MQTT operation.
+ *
+ * @param[in] pOperation The completed MQTT operation.
+ *
+ * @return `IOT_MQTT_SUCCESS` if the schedule was successful;
+ * `IOT_MQTT_SCHEDULING_ERROR` otherwise.
+ */
+static IotMqttError_t _scheduleCallback( _mqttOperation_t * pOperation );
+
+/**
+ * @brief Complete a pending send operation.
+ *
+ * @param[in] pOperation The pending MQTT send operation.
+ * @param[out] pDestroyOperation Whether the operation should be destroyed afterwards.
+ *
+ * @return `true if the operation is awaiting a response from the network;
```

Comment:
You're missing the trailing backtick for true. I don't think the backticks are necessary in general, though.

Reply:
Thanks, will fix. I'm not sure about the backticks, but I want to be consistent with the other comments in this file.

Reply:
The backticks are used so that variable names show up `like this` in the generated documentation.